src/components/common/TargetListItem.js

```diff
@@ -15,6 +20,21 @@ const TargetListItem = ({ title, topic, lat, lon, setMapCenter }) => {
         });
       }}
     >
+      <div className="target-item__trash-button-container">
+        <button
+          className="target-item__trash-button"
+          onClick={() => {
+            dispatch(targetActions.deleteTarget(id));
+          }}
+        >
+          <img
+            className="target-item__trash-button-img"
+            type="image"
```

Comment:
what would be the function of this **type** prop here?

Reply:
It's an error, I'll delete it